### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values/questlist.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#201]
```
English proofread.  Typos corrected, a few grammatical changes :)
```

Original issue reported on code.google.com by `medic....@gmail.com` on 13 Jun 2011 at 6:26

Attachments:
- [questlist.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-201/comment-0/questlist.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Committed earlier. I forgot which revision though :) 
```

Original comment by `oskar.wi...@gmail.com` on 22 Jun 2011 at 8:24
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.9**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:53
- Changed state: **Done**
